samples/basic-order-processing-client-nestjs/src/app.module.ts

```diff
@@ -0,0 +1,23 @@
+import { AmqpClientModule } from "@amqp-contract/client-nestjs";
+import { Module } from "@nestjs/common";
+import { OrderService } from "./order.service.js";
+import { orderContract } from "@amqp-contract-samples/basic-order-processing-contract";
+import { z } from "zod";
+
+const env = z
+  .object({
+    AMQP_URL: z.string().url().default("amqp://localhost:5672"),
+  })
+  .parse(process.env);
```

Comment:
perfect, you validate the env using a zod schema. do it in a `ConfigModule.forFeature` in order to have a provider nestjs with the config